### PR TITLE
Series of small improvements

### PR DESCRIPTION
**Pull request by terceiro**

### CODE REVIEW

File: frontend/static/templates/build_detail.html [hunk 1]
```diff
@@ -110,6 +110,11 @@ <h1 class="md-display-1"> <small>{{ build.created_at }}</small><br/>#{{ build.bu
       <div class="form-group">
         <input type="text" ng-model="queryBenchmarks" class="form-control input-sm" placeholder="filter">
       </div>
+      <div class='checkbox'>
+        <input id='show-boring' type='checkbox' ng-model='showBoring'/>
+        <label for='show-boring'>Show "uninteresting" results</a>
```
Comment (mwasilew) [inline frontend/static/templates/build_detail.html]: where is an opening tag for </a> ?

**Comment by mwasilew** [comment]
Looks good. With the last commit (the uninteresting results) I had 3 checkboxes in mind - 1 for each category of results. However this also looks useful. I would change the label to 'insignificant'.

**Comment by mwasilew** [comment]
I also applied this:

```
diff --git a/frontend/static/main.css b/frontend/static/main.css
index 3d7e499..94c5d8e 100644
--- a/frontend/static/main.css
+++ b/frontend/static/main.css
@@ -104,5 +104,5 @@ input, select, button, .btn {
 }
 
 #charts {
-    min-height: 550px;
+    min-height: 750px;
 }
diff --git a/frontend/static/main.js b/frontend/static/main.js
index b761ab2..f935356 100644
--- a/frontend/static/main.js
+++ b/frontend/static/main.js
@@ -339,6 +339,9 @@ app.controller('Stats', ['$scope', '$http', '$routeParams', '$timeout', '$q', '$
 
             Highcharts.chart(
                 document.getElementById('charts'), {
+                    chart: {
+                        zoomType: "xy"
+                    },
                     title: {
                         text: $scope.benchmark.label + ' on branch ' + $scope.branch.branch_name + '; ' + envs
                     },

```
Makes the chart even more useful. 750px height works on my smallest screen.

**Comment by terceiro** [comment]
all fixed - although I preferred 'non-significant' instead of 'insignificant' as the former sounds more appropriate from a statistics PoV

**Pull request merged**